### PR TITLE
TESTNET 30 - fixed issues in smart Contracts and write some test...

### PR DESCRIPTION
**Pull request by Akhilleshgoswami**
# Description :world_map:
install Imp npm packages, update the package.json,  update truffle Config,  fixed sc issues, create a migration file, and write some tests.

Fixes # (issue) :rocket:
* solidity version  .
* imports statement .

# How to perform the test? :rocket:
1. you need a truffle to install on your PC.
2. run truffle develop.
3. truffle migrate --reset.
4. copy the contract address and put that in the USDCPool.test.js file in the place of the contract address.
6. run truffle test 

#  WIP :hourglass:
writing test for smart contract.

**Test Configuration**:
* OS: Linux 
* Framework: truffle V5.0.1.
* language: solidity ^0.8.0, javascript

### CODE REVIEW

File: test/USDCPoll.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  describe("Contract Function testing", async () => {
+    it("provide liquidity", async () => {
+      let supply = await USDCPool.totalSupply();
+      let balance = await USDCPool.totalBalance();
+      balance = balance.toNumber();
+      supply = supply.toNumber();
+      console.log((balance += 1));
+      console.log(supply);
+      console.log(balance);
+      let USDC_Token_value = Math.floor((1 * supply) / (balance - 1));
```
Comment (Prithviraj8) [inline test/USDCPoll.test.js:41]: sorry what is this formula here? Its computing the USDC token value/price ?

File: truffle-config.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ module.exports = {
   // Configure your compilers
   compilers: {
     solc: {
-      // version: "0.5.1",    // Fetch exact version from solc-bin (default: truffle's version)
+       version: "^0.8.0",    // Fetch exact version from solc-bin (default: truffle's version)
```
Comment (Prithviraj8) [inline truffle-config.js:84]: can remove all the unrelated comments from this file.

File: contracts/USDCPool.sol [hunk 3]
```diff
@@ -105,7 +105,7 @@ contract USDCPool is
      * @nonce calls by CruizePutOptions to lock the premiums
      * @param amount Amount of premiums that should be locked
      */
-    function sendPremium() external override payable onlyOwner {
+    function sendPremium() external  payable onlyOwner {
```
Comment (Prithviraj8) [inline contracts/USDCPool.sol:108]: The space between "external" & "payable" has 1 extra space 😛. Sorry its my OCD

File: test/USDCPoll.test.js (hunk 1 above)
Comment (Akhilleshgoswami) [reply test/USDCPoll.test.js:41]: yes right

File: contracts/USDCPool.sol (hunk 3 above)
Comment (Akhilleshgoswami) [reply contracts/USDCPool.sol:108]: ohk  done

File: truffle-config.js (hunk 2 above)
Comment (Prithviraj8) [reply truffle-config.js:84]: okay cool then don't remove for now.

**Pull request closed without merging**